### ISSUE TITLE
The _NFP suffix of some lemmas. 

### ISSUE THREAD

**Issue by foxik** [UniversalDependencies/UD_English#3]
Some lemmas with NFP part of speech have _NFP suffix, for example see first 20 NFP tokens in the training data:
```
1   ---------  ---------_NFP  PUNCT  NFP  _  0   root       _  _
1   ***        ***            PUNCT  NFP  _  0   root       _  _
1   +          +              SYM    NFP  _  17  punct      _  _
60  +          +              SYM    NFP  _  55  punct      _  _
9   -          -_NFP          PUNCT  NFP  _  11  punct      _  _
1   ...        ...            SYM    NFP  _  0   root       _  _
1   ...        ...            SYM    NFP  _  0   root       _  _
1   ...        ...            SYM    NFP  _  0   root       _  _
1   ...        ...            SYM    NFP  _  0   root       _  _
9   :-RRB-     :-rrb-_NFP     SYM    NFP  _  5   discourse  _  _
1   -          -_NFP          PUNCT  NFP  _  3   punct      _  _
9   :-RRB-     :-rrb-_NFP     SYM    NFP  _  3   discourse  _  _
1   =          =              SYM    NFP  _  3   punct      _  _
1   ==         ==             SYM    NFP  _  5   discourse  _  _
20  ***        ***            PUNCT  NFP  _  21  punct      _  _
22  ***        ***            PUNCT  NFP  _  21  punct      _  _
18  :-RRB-     :-rrb-_NFP     SYM    NFP  _  3   discourse  _  _
1   *          *              PUNCT  NFP  _  2   punct      _  _
12  *          *              PUNCT  NFP  _  2   punct      _  _
1   *          *              PUNCT  NFP  _  2   punct      _  _
```
Is this deliberate, or it is some kind of error or compatibility issue?

Personally I would drop the `_NFP` suffix of lemmas.

EDIT: I updated the example to use -RRB- instead of ), I performed grep on wrong branch.

**Comment by manning** [comment]
Hi @foxik, this is indeed an error, thanks!  "NFP" is a new Penn treebank POS tag for Non-Final Punctuation, and I guess something is going wrong with our pipeline on it, and it's living on in the lemmas. This should be fixed.

**Comment by manning** [comment]
Fixed in our version 1.2 release candidate.
